configuration/micro_kernel_trait.rst

```diff
@@ -350,4 +350,4 @@ Then see webpage in browser:
 
 Hey, that looks a lot like a *traditional* Symfony application! You're right: the
 ``MicroKernelTrait`` *is* still Symfony: but you can control your structure and
-features quite easily.
+features in a lighter way.
```

Comment:
how about "with less boilerplate configuration and code."? 

`a lighter way` seems less meaningful to me than the `quite easy` (which is belittling language, but transports the intention of this).